deployments/ansible/roles/collector/README.md

```diff
@@ -4,6 +4,24 @@ Ansible role that installs Splunk OpenTelemetry Connector configured to
 collect metrics, traces and logs from Linux machines and send data to [Splunk 
 Observability Cloud](https://www.splunk.com/en_us/observability.html). 
 
+## Linux
+Currently, the following Linux distributions and versions are supported:
+
+- Amazon Linux
+- RedHat
+- Debian: buster, stretch
+- Ubuntu: 16.04, 18.04, 20.04
+
+## Windows
+Currently, the following Windows versions are supported:
+- Ansible requires PowerShell 3.0 or newer and at least .NET 4.0 to be installed on Windows host.
+A WinRM listner should be created and activated. 
+For setting up Windows Host Refer : [Ansible Docs](https://docs.ansible.com/ansible/latest/user_guide/windows_setup.html)
+
+- Windows Server 2012 64-bit
+- Windows Server 2016 64-bit
+- Windows Server 2019 64-bit
```

Comment:
Can we move this list higher right under "Currently, the following Windows versions are supported:" and separate it from the list of Windows requirements